src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -189,16 +191,25 @@ private static bool IsInvalidDouble(double number)
             });
         }
 
-        // Wraps a scalar function into its tabular overload
-        private static AsyncFunctionPtr StandardErrorHandlingTabularOverload<TScalar>(string functionName, AsyncFunctionPtr targetFunction)
+        /// <summary>
+        /// Wraps a scalar function into its tabular overload.
+        /// </summary>
+        /// <param name="functionName">Function name.</param>
+        /// <param name="targetFunction">Target function to execute.</param>
+        /// <param name="replaceBlankValues">Only supply this if its scaler function has <see cref="NoOpAlreadyHandledByIR(IRContext, int)"/>, meaning scaler has handled this via IR.</param>
```

Comment:
>scaler [](http://example.com/codeflow?start=69&length=6)

[typo] scalar. Same issue in other places of this file

Reply:
oops! Thanks, for pointing that out!